cfpb/cfgov-refresh
# Output field required note in Spanish if page language is Spanish

We're about to launch our first (I think) page in Spanish with an email signup form. This PR makes it so that the "(required)" note on a Form Field with Button molecule will appear in Spanish as "(se require)" if the page language is set to Spanish in its Configuration tab.

## Additions

- Template code to check whether a required form field is in English or Spanish and output the required note accordingly

## Testing

1. Pull branch
2. Run server
3. Preview a page with an email sign-up module, like http://localhost:8000/owning-a-home/
4. Observe that the "(required)" note is output as expected in English.
5. Edit that page and change its language to Spanish, then preview again.
6. Observe that the "(se require)" note is output as expected in Spanish.

## Screenshots

![screen shot 2017-08-29 at 13 27 17](https://user-images.githubusercontent.com/1044670/29834758-cf586832-8cbd-11e7-9116-6aa174e4ac6a.png)

## Notes

- Need to revisit the class-based hiding/showing of alternate language messages being used on the Share and Expandable modules to see if we can check `page.language` there, as well. I feel like it may not have been possible on those templates for some reason, but I will double-check.

## Todos

- Add a test for this logic?

## Checklist
